modules/core/src/test/java/org/apache/ignite/internal/processors/cache/GridCacheUtilsSelfTest.java

```diff
@@ -104,7 +121,8 @@ public boolean equals(String obj) {
 
     /**
      */
-    public void testCacheKeyValidation() {
+    @SuppressWarnings("ResultOfMethodCallIgnored")
+    public void testCacheKeyValidation() throws IgniteCheckedException {
```

Comment:
Let's add the test and support custom keys that are essentially binary objects. Refer to one of my review note above.